server/scripts/debug_defoldsdk.py

```diff
@@ -2,24 +2,24 @@
 
 import os
 import sys
-import urllib
+import urllib, urllib.request
 import json
 import shutil
 import argparse
 
 def get_latest_version():
     url = "http://d.defold.com/stable/info.json"
-    response = urllib.urlopen(url)
+    response = urllib.request.urlopen(url)
     if response.getcode() == 200:
         return json.loads(response.read())
     return {}
 
 def download_file(url):
 	print("Downloading %s" % url)
 	tmpfile = '_dl.zip'
-	cmd="wget -O %s %s" % (tmpfile, url)
-	result = os.system(cmd)
-	if result:
+	try:
+		urllib.request.urlretrieve(url, tmpfile)
```

Comment:
`urlretrieve` doesn't have explicit HTTP response code. In case of 404 - it throws an exception.